src/main/java/org/jabref/model/groups/LastNameGroup.java

```diff
@@ -0,0 +1,59 @@
+package org.jabref.model.groups;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.jabref.model.entry.Author;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+
+/**
+ * Matches based on a latex free last name in a specified field. The field is parsed as an author list and the last names are resolved of latex.
+ */
+public class LastNameGroup extends KeywordGroup {
+    public LastNameGroup(String groupName, GroupHierarchyType context, Field searchField, String lastName) {
+        super(groupName, context, searchField, LatexToUnicodeAdapter.format(lastName), true);
+    }
+
+    static List<String> getAsLastNamesLatexFree(Field field, BibEntry bibEntry) {
+        return bibEntry.getField(field)
+                       .map(AuthorList::parse)
+                       .map(AuthorList::getAuthors)
+                       .map(authors ->
```

Comment:
if you add flatmap already here, it allows to chain everything nicely I think

Reply:
... I forgot that `Optional.stream` existed... 🤯